File: med-app-backend-spring/src/main/java/com/medapp/app/dts/medappbackendspring/Dto/GetDoctorAdById.java
package com.medapp.app.dts.medappbackendspring.Dto;

import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class GetDoctorAdById {
    private Long id;
    private String title;
    private String category;
    private String address;
    private String description;
    private String city;
    private Long price;
    private String rating;
    private DoctorMainInfo doctor;
}

File: med-app-android/app/src/main/java/dts/app/med_app_android/Model/GetDoctorAdById.kt
package dts.app.med_app_android.Model

data class GetDoctorAdById(
    val address: String,
    val category: String,
    val description: String,
    val city: String,
    val doctor: DoctorMainInfo,
    val id: Long,
    val price: Long,
    val rating: Double,
    val title: String
)